torch/csrc/api/src/xpu.cpp

```diff
@@ -0,0 +1,19 @@
+#include <torch/xpu.h>
+
+#include <ATen/Context.h>
+
+#include <cstddef>
```

Comment:
Why this is needed (already included from xpu.h isn't it?)

Reply:
Removed.